docs/reference/release-notes/highlights.asciidoc

```diff
@@ -37,4 +37,90 @@ Other versions:
 // [discrete]
 // === Heading
 //
-// Description.
\ No newline at end of file
+// Description.
+
+// tag::notable-highlights[]
+
+[discrete]
+[[frozen-tier-ga]]
+=== Frozen tier is now GA
+
+The frozen tier and shared-cache searchable snapshots are now generally
+available. These features were introduced as technical previews in 7.12. See
+the
+{ref-bare}/7.12/release-highlights.html#frozen-tier-shared-snapshot-cache[7.12
+announcement].
+
+[discrete]
+[[index-runtime-fields]]
+=== Index runtime fields
+
+Runtime fields let you dynamically create fields from other fields and document
+properties at query time. These query-time runtime fields optimize for
+flexibility over speed, letting you change them at any time.
+
+With {minor-version}, you can now also create index-time runtime fields, trading
+flexibility for better query performance. See {ref}/runtime-indexed.html[Index a
+runtime field].
+
+[discrete]
+[[match-ipv4-ipv6-addresses-against-cidr-ranges-painless]]
+=== Match IPv4 and IPv6 addresses against CIDR ranges in Painless
+
+Painless expressions can now match IPv4 and IPv6 addresses against CIDR ranges.
+This is useful for classifying and grouping IP addresses in security and
+monitoring use cases. This feature is available in all contexts, including
+runtime fields, ingest, search, and aggregations.
+
+For example:
+
+[source,painless]
+----
+CIDR myCidrRange = new CIDR('10.1.1.0/25');
+myCidrRange.contains('10.1.1.127'); // true
+myCidrRange.contains('10.1.1.129'); // false
+
+myCidrRange = new CIDR('2001:0db8:85a3::/64');
+myCidrRange.contains('2001:0db8:85a3:0000:0000:8a2e:0370:7334'); // true
+myCidrRange.contains('2001:0db8:85a3:0001:0000:8a2e:0370:7334'); // false
+----
+
+[discrete]
+[[automatic-updates-geoip-processor]]
+=== Automatic updates for the GeoIP processor
+
+By default, {es} now automatically downloads GeoIP2 database updates for the
+`geoip` processor from the Elastic GeoIP endpoint:
+https://geoip.elastic.co/v1/database. You can use the new
+{ref}/geoip-stats-api.html[GeoIP stats API] to get download stats for these
+updates. For more information, see the updated
+{ref}/geoip-processor.html[`geoip` processor documentation].
+
+[discrete]
+[[new-combined-fields-query-type]]
+=== New `combined_fields` query type
+
+7.13 introduces the `combined_fields` query, new a Query DSL query type for
+searching multiple `text` fields as a combined field. You can use the
+`combined_fields` query as a simpler alternative to the `multi_match` query's
+`cross_fields` type option. See the
+{ref}/query-dsl-combined-fields-query.html[`combined_fields` query
+documentation].
+
+[discrete]
+[[faster-terms-aggregations]]
+=== Faster `terms` aggregations
+
+We've improved the speed of the `terms` aggregation by up to 92% when:
```

Comment:
"up to" isn't right - there isn't really a max. If all these hold true then we return the answer in nearly constant time. Its not *quite* constant. If you have more segments and we have to sum up segment metadata. But it's pretty darn low. You can basically make the index as big as you like and it'll still take 30ms to answer.

Reply:
Thanks for catching this @nik9000. I've updated this to just state:

`We've significantly improved the speed of the `terms` aggregation when: ...`